internal/cephfs/controllerserver.go

```diff
@@ -823,6 +819,40 @@ func (cs *ControllerServer) CreateSnapshot(
 	// too.
 	volClient := core.NewSubVolume(parentVolOptions.GetConnection(), &parentVolOptions.SubVolume,
 		parentVolOptions.ClusterID, cs.ClusterName, cs.SetMetadata)
+
+	if parentVolOptions.BackingSnapshot {
+		// update the reference count for parent volume
+		if err = store.UpdateRefCount(ctx, parentVolOptions); err != nil {
+			return nil, status.Error(codes.InvalidArgument, "failed to snapshot a snapshot-backed volume")
+		}
+		// Reservation
+		sID, er := store.ReserveSnap(ctx, parentVolOptions, vid.FsSubvolName, cephfsSnap, cr)
+		if er != nil {
+			return nil, status.Error(codes.Internal, er.Error())
+		}
```

Comment:
Have you checked the reference tracker object before and after this operation ?
Can you paste the output?

Reply:
Checking now.